vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/TouchSteps.java

```diff
@@ -152,6 +166,30 @@ public void swipeToElement(SwipeDirection direction, Locator locator, Duration s
         }
     }
 
+    /**
+     * Presses the key
+     * <br>
+     * See <a href="https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton">iOS keys</a> and
+     * <a href="https://appium.github.io/java-client/io/appium/java_client/android/nativekey/AndroidKey.html">
+     * Android keys</a> for available values
+     * 
+     * @param key key to press
+     */
+    @When("I press $key key")
```

Comment:
I would say it shouldn't be in `TouchSteps`, rather `DeviceSteps` should be used

Reply:
Why? not all keys are physical, we can tap on input field and then use this step for typing